src/components/Navigation.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
-const Navigation = () => {
+const Navigation = ({ confirmDeleteUserToken }) => {
   return (
     <nav>
+      <NavLink to="/" activeClassName="active-link">
```

Comment:
Try adding the `exact` prop to the NavLink...it seems like then it will only be active if it matches the path exactly. https://reactrouter.com/web/api/NavLink/exact-bool

Reply:
Thanks for that Lauren. The active class works as intended now 😄